spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/oauth2/client/OAuth2SsoProperties.java

```diff
@@ -41,6 +44,88 @@
 	 */
 	private Integer filterOrder;
 
+	/**
+	 * If <code>true</code>, will always redirect to the value of {@link #succesDefaultTargetUrl}
+	 * (defaults to <code>false</code>).
+	 *
+	 * @see SavedRequestAwareAuthenticationSuccessHandler#setAlwaysUseDefaultTargetUrl(boolean)
+	 */
+	private boolean successAlwaysUseDefaultTargetUrl = false;
```

Comment:
As [described in the documentation](http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#configuration-metadata-annotation-processor) you can't use complex javadoc tag for field Javadoc of `@ConfigurationProperties` annotated bean as we use it as a source for the documentation that tools use for auto-completion.